pluginconfig/commit.go

```diff
@@ -104,9 +72,25 @@ type CommitOffchainConfig struct {
 	// This will typically be an arbitrum testnet/mainnet chain depending on
 	// the deployment.
 	TokenPriceChainSelector uint64 `json:"tokenPriceChainSelector"`
+
+	// The maximum number of times to check if the previous report has been transmitted
+	MaxReportTransmissionCheckAttempts uint `json:"maxReportTransmissionCheckAttempts"`
+
+	// SyncTimeout is the timeout for syncing the commit plugin reader.
+	SyncTimeout time.Duration `json:"syncTimeout"`
```

Comment:
If we don't use defaults for these let's add them validation for them.

Reply:
we have multiple readers and better clarify here.

Reply:
We have defaults for this that are applied in the plugin constructors. I'm wondering if there should be a standard place where defaults are applied. In the plugin constructor seems fine to me at the moment though.